src/instructlab/sdg/generate_data.py

```diff
@@ -139,13 +139,33 @@ def _gen_train_data(
             outfile.write("\n")
 
 
+def _knowledge_seed_example_to_test_data(seed_example):
+    res = []
+    for qna in seed_example["questions_and_answers"]:
+        user = qna["question"] + "\n" + seed_example["input"]  # context
```

Comment:
`seed_example["input"]` looks wrong - it should be `seed_example["context"]` ?

According to `instructlab/schema/v3/knowledge.json`, items in `seed_examples` have 2 properties - `context` and `questions_and_answers`

Reply:
Ok, here's where `context` gets renamed to `input` while reading the taxonomy files:

```
        for seed_example in contents.get("seed_examples"):
            context = seed_example.get("context", "")
            if "questions_and_answers" in seed_example:
                question_answer_list = seed_example.get("questions_and_answers")
                seed_instruction_data.append(
                    {
                        "questions_and_answers": question_answer_list,
                        "input": context,
```

That seems unnecessarily confusing to me

Reply:
Fixed in commit d0df58dc707a13344b8c4ac0e4a80c660897250f